### ISSUE TITLE
Accessibility Support from Benefits Decision Review Team

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#71952]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits Decision Review
- **OCTO product owner:** Zach Goldfine
- **Product name:** Board Appeal / Notice of Disagreement
- **Product manager:**  Sade Ragsdale
- **Slack channel:** [Benefits-Decision-Review](https://dsva.slack.com/archives/C5AGLBNRK)
- **Dedicated content writer on your team (if you have one):** N/A - Designer - Eileen Chi
- **Dedicated a11y specialist on your team (if you have one):** DK, David Kennedy
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else:  adding a new HTML page to the end of appeal forms

### What's the nature of your initiative and desired outcomes?

Today, when Veterans file an appeal on va.gov, there is no way for them to maintain a record of their submission. To increase Veteran trust of the reliability of our products, we want to provide the ability to review and/or download a PDF copy of a submitted appeal, so veterans can store it in their personal records.

[Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Notice-of-Disagreement/Appeal-PDF-Download-Initiative%20Brief.md)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Working on Research Plan and Conversation Guide now

Code prototype will be done next sprint

Hoping to conduct usability testing early-mid January

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #66135 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Notice-of-Disagreement/Appeal-PDF-Download-Initiative%20Brief.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691599300063/42cd59780f131a0a61b149f7df8aa9c7f8ae83f6?wid=0-1701439918010&outline=open)
- [Prototype or mockups](https://www.sketch.com/s/0424c51d-7818-43ea-8821-6de91eb7ab27/prototype/40A84FC0-A2EC-46CF-8DAB-03E58C1D7934/a/40A84FC0-A2EC-46CF-8DAB-03E58C1D7934)
- Research plan) In progress
- Research conversation guide In progress
- [Epics/issue that may be helpful](https://app.zenhub.com/workspaces/benefits-team-1-6138d7b57a2631001a4b7562/issues/gh/department-of-veterans-affairs/va.gov-team/71358)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
```

```[tasklist]
### CAIA a11y tasks
- [ ] A11y task 1
- [ ] A11y task 2
```

```[tasklist]
### CAIA content tasks
- [ ] Content task 1
- [ ] Content task 2
```

```[tasklist]
### CAIA IA tasks
- [ ] IA task 1
- [ ] IA task 2
```

```[tasklist]
### CAIA operations
- [ ] Set up kickoff meeting - @coforma-terry and @strelichl 
- [ ] Provide GitHub deliverable to VFS team post kickoff
```

**Comment by eileen-coforma** [comment]
Our A11y lead @davidakennedy would also be able to sit in some of the sessions, so we may not need full support for the study.

**Comment by coforma-terry** [comment]
Thank you @eileen-coforma! Let us know :)

**Comment by sara-amanda** [comment]
@coforma-jamie @eileen-coforma @davidakennedy @coforma-terry and Evan (no GH yet) `a11y task list` and `VFS task list` have been updated to reflect our meeting today. Thanks again for meeting with us!

**Comment by sara-amanda** [comment]
## Pilot Scheduling

Hi @eileen-coforma [re: Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1705590944581969)

> @eileen-coforma - that is great news that you are getting ready for research!
> So, yes, like @coforma-terry mentioned, we have quite a few studies taking place [on our capacity calendar during that timeframe](https://calendar.google.com/calendar/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a%40group.calendar.google.com&ctz=America%2FNew_York) ...
> 
> - Would you be able to do a pilot the week of 2/5?
>     - Perhaps, on Wednesday-Friday of that week?
> - We would need a research ticket.
>     - The ticket template for research tickets can be found in your intake ticket:
>         - See Tasks (Next Steps) >
>             - There is a link to the template (I will also paste it here for ya):
>                 - [A11y research support request (template)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry,angelafowler82&labels=508/Accessibility,%20sitewide%20CAIA,%20sitewide%20accessibility,%20CAIA-a11y-research&template=caia-a11y-research.yaml&title=[CAIA%20A11y%20Research]:%20Team%20name,%20product%20name)

**Comment by eileen-coforma** [comment]
Yes, thank you for your help!
